Microsoft.Toolkit.Uwp.UI.Controls/TabbedCommandBar/TabbedCommandBarItem.cs

```diff
@@ -0,0 +1,118 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// A <see cref="CommandBar"/> to be displayed in a <see cref="TabbedCommandBar"/>
+    /// </summary>
+    [TemplatePart(Name = "PrimaryItemsControl", Type = typeof(ItemsControl))]
+    [TemplatePart(Name = "MoreButton", Type = typeof(Button))]
+    public class TabbedCommandBarItem : CommandBar
+    {
+        private ItemsControl _primaryItemsControl;
+        private Button _moreButton;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TabbedCommandBarItem"/> class.
+        /// </summary>
+        public TabbedCommandBarItem()
+        {
+            DefaultStyleKey = typeof(TabbedCommandBarItem);
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="Header"/> property.
+        /// </summary>
+        public static readonly DependencyProperty HeaderProperty = DependencyProperty.Register(
+            nameof(Header),
+            typeof(string),
```

Comment:
Since `NavigationViewItem.Content` is an object, our `Header` here should be an `object` as well. That way if someone wants to customize it and add an icon or something next to their text they can pass that in XAML with a different control.